src/Skeleton/Skeleton.scss

```diff
@@ -0,0 +1,3 @@
+.react-loading-skeleton {
+    z-index: unset;
```

Comment:
`react-loading-skeleton` puts `z-index: 1` on their skeletons. This negatively impacted the shadow layering within a `FullscreenModal` and `Stepper`. Unsetting removes the styling from `react-loading-skeleton`.